consultation_analyser/consultations/models.py

```diff
@@ -1,7 +1,43 @@
-# Create your models here.
 from django.db import models
 
 
+class Consultation(models.Model):
+    # May need something better long-term
+    class Organisation(models.TextChoices):
+        DFE = "dept-for-education", "Department for Education"
+        CO = "cabinet-office", "Cabinet Office"
+
+    identifier = models.CharField(max_length=256, blank=False)
+    name = models.CharField(max_length=256, blank=False)
+    organisation_identifier = models.CharField(max_length=256, choices=Organisation.choices, blank=True)
+
+
 class Question(models.Model):
     text = models.CharField(max_length=None)  # no idea what's a sensible value for max_length
     slug = models.CharField(max_length=256)
+    is_closed = models.BooleanField(default=False)
+    is_open = models.BooleanField(default=False)
+    closed_response_options = models.JSONField(default=list)
+    order = models.IntegerField(blank=True, null=True)
+    consultation = models.ForeignKey(Consultation, on_delete=models.CASCADE, null=True)
+
+
+class Respondent(models.Model):
```

Comment:
I see that we need this to group the answers, but maybe we could leave out the characteristics for now as we don't know much about them yet